include/uapi/sound/sof/abi.h

```diff
@@ -25,8 +25,8 @@
 #define __INCLUDE_UAPI_SOUND_SOF_ABI_H__
 
 /* SOF ABI version major, minor and patch numbers */
-#define SOF_ABI_MAJOR 3
```

Comment:
well no, this isn't a major ABI change. It's likely even backwards compatible as is, no?